turbo/cli/flags.go

```diff
@@ -250,6 +250,15 @@ func ApplyFlagsForEthConfig(ctx *cli.Context, cfg *ethconfig.Config) {
 		}
 	}
 
+	if ctx.Bool(utils.DisableIPV6.Name) {
+		log.Info("Downloader Disabled IPV6")
+		cfg.Downloader.ClientConfig.DisableIPv6 = true
+	}
+
+	if ctx.Bool(utils.DisableIPV4.Name) {
+		log.Info("Downloader Disabled IPV4")
+		cfg.Downloader.ClientConfig.DisableIPv4 = false
```

Comment:
1. probably here must be `true`
2. let's write exactly same style log line as we do in ./cmd/downloader and in ./cmd/erigon (about downloader component configuration): 
1-line which describe all params: ipv4,ipv6,uploadrate,downloadrate,...

Reply:
Done!